packages/cozy-harvest-lib/src/helpers/fields.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Prepare props to pass to React-Select
+ * Options must be consistent and value cannot be ''
+ * @param  {props} props Props passed to Cozy-UI <Field /> component
+ * @return {[type]}       Sanitized props, ready to be passed to
+ */
+export const sanitizeSelectProps = props => {
+  const { options, value } = props
+  const sanitized = { ...props }
+
+  sanitized.options = options
+    ? options.map(option => ({
+        ...option,
+        // legacy
+        label: option.name
```

Comment:
For non legacy field we will overwrite the `label` prop here?

Reply:
Well spotted, `!option.label` && option.name` should be better.

Reply:
Update : `option.label || option.name` is faaaaar better.